app/assets/stylesheets/events.scss

```diff
@@ -10,66 +10,77 @@
 }
 
 .epage {
-	background: asset_url("homepage-pic.jpg") no-repeat;
-	background-size: 100%;	
-	height: 665px;
+	background: asset_url("mainpages/eventspage.jpg") no-repeat center center fixed;
+	-webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  height: 100%;
 }
 
 .transbox {
-	height: 350px;
-	margin: 30px;
-  background-color: #ffffff;
-  opacity: 0.5;
-  filter: alpha(opacity=50); /* For IE8 and earlier */
+	// height: 350px;
```

Comment:
Since this whole thing is commented out can we remove it? Or will it be needed later?